Explain public streams and filters (#449)
dolittle/Runtime: Documentation/Concepts/event_handlers_and_filters.md

However with great power comes great responsibility. The filters cannot be changed in a way so that it breaks the [rules of streams]({{< ref "#rules" >}}). If it does, the Runtime would notice it and return a failed registration response to the head that tried to register the filter.

### Public Filters

> Since there are [two types of streams]({{< ref "streams.md#public-streams" >}}) there needs to be two different kinds of filters; public and private. The private filter is explained above, and the public filter functions exactly the same way. The only difference is that it creates a [public stream]({{< ref "streams.md#public-streams" >}}).

Reviewer: ```suggestion
Since there are [two types of streams]({{< ref "streams.md#public-streams" >}}) there are two kinds of filters; public and private. They function in the same way, except that a private filters creates private streams and a public filter creates [public streams]({{< ref "streams.md#public-streams" >}}).
```
Removed the "above" line, just in case we move the articles around someday. Reworked the sentence a bit and fixed a typo.
